provisioning_templates/provision/kickstart_default.erb

```diff
@@ -52,7 +52,8 @@ skipx
 <% else -%>
 <% dhcp = !@static -%>
 <% end -%>
-network --bootproto <%= dhcp ? 'dhcp' : "static --ip=#{@host.ip} --netmask=#{subnet.mask} --gateway=#{subnet.gateway} --nameserver=#{[subnet.dns_primary, subnet.dns_secondary].select{ |item| item.present? }.join(',')}" %> --hostname <%= @host %><%= os_major >= 6 ? " --device=#{@host.mac}" : '' -%>
+
+network --bootproto <%= dhcp ? 'dhcp' : "static --ip=#{@host.ip} --netmask=#{subnet.mask} --gateway=#{subnet.gateway} --nameserver=#{[subnet.dns_primary, subnet.dns_secondary].select{ |item| item.present? }.join(',')} --mtu=#{subnet.mtu.to_s}" %> --hostname <%= @host %><%= os_major >= 6 ? " --device=#{@host.mac}" : '' -%>
```

Comment:
shouldn't the MTU get set always here?

Reply:
@mmoll, as described in [RFC2132](https://www.ietf.org/rfc/rfc2132.txt), the MTU can be set via DHCP.

eg, for ISC DHCPD:
```
option interface-mtu 9000;
```

So I think that if a dhcp server is used to provide network configuration for hosts on a network, the MTU will be set via the dhcp server too.

Reply:
I see, let's bring this in then :+1: